client/src/components/Panels/Side/Right/Test/Test.tsx

```diff
@@ -20,7 +24,50 @@ const Test = () => {
 
   const { initialLoading, deployed, connError } = useInitialLoading();
 
-  const idl = PgProgramInfo.getProgramInfo()?.idl;
+  const idl = useMemo(() => PgProgramInfo.getProgramInfo()?.idl, []);
```

Comment:
This change will result in IDL not getting updated if you `build` from the terminal when the test section is visible. You can either remove the `useMemo` or use `buildCount` as a depeendency from
```ts
const [buildCount] = useAtom(buildCountAtom)
```

I think it's probably better to not use `useMemo` at all because the parent component is not getting rendered a lot anyway but I'm open to both changes.

Reply:
Thanks, I've removed the `useMemo` and checked that it updates now when you run `build` in the terminal.